src/main/scala/com/gu/octopusthrift/models/OctopusArticle.scala

```diff
@@ -0,0 +1,108 @@
+package com.gu.octopusthrift.models
+
+import java.util.concurrent.TimeUnit
+
+import com.gu.flexibleoctopus.model.thrift._
+import org.joda.time.format.DateTimeFormat
+import org.joda.time.{ DateTime, DateTimeZone }
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+import scala.util.{ Success, Try }
+
+/**
+ * This model does not represent all fields sent by Octopus,
+ * only those that will be relevant in converting to our Thrift model
```

Comment:
minor: Is it worth linking to where one can find the full model? If it exists.

Reply:
The test json blobs go some way to providing an exhaustive list, maybe that's enough.

Reply:
I don't believe we currently have a full model definition, outside of our own examples and documentation. I think we considered modelling it in it's entirety, but there's a good deal of data on the original model that has no bearing on any of our filtering logic or on the shape of the final Thrift payload.